src/python/services/queue-service/queue_service.py

```diff
@@ -0,0 +1,63 @@
+from azure.identity import AzureCliCredential
+from azure.storage.queue import QueueServiceClient
+from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.cosmos import CosmosClient
+
+import requests
+import os
+import time
+import json
+
+from dotmap import DotMap
+from dotenv import load_dotenv
+
+load_dotenv("../../.env")
```

Comment:
nit: Consider experimenting with `find_dotenv` to reduce hard-coded path in this module ...i.e. 
```
from dotenv import load_dotenv, find_dotenv
load_dotenv(find_dotenv())
```

Another pattern pivot is to have the dotenv interaction stuff in a different module like `settings.py` and import it into this one.

Reply:
Done